### ISSUE TITLE
Add IncludeGdc in JsonLayout

### ISSUE THREAD

**Issue by casperc89** [NLog/NLog#3049]
**NLog version**: 4.5

**Platform**: .Net 3.5 / .Net 4.0 / .Net 4.5 / Mono 4 / Silverlight 4 / Silverlight 5 / Xamarin Android / Xamarin iOs / .NET Core 1 / .NET Core 2

**Why do we need it?**
JsonLayout currently has support to log all properties from `MappedDiagnosticsContext` & `MappedDiagnosticsLogicalContext` through  `IncludeMdc` & `IncludeMdlc`. Unfortunately the feature to include properties from `GlobalDiagnosticsContext` is missing. 

Since we're using these properties in our application through a generic template. It would be awesome to have this feature for the `GlobalDiagnosticsContext` as well.

**An example of the XML config, if relevant.**
```xml
<layout type=""JsonLayout"" IncludeMdc='true' IncludeMdlc='true' IncludeGdc='true' >...</layout>
```

**Comment by 304NotModified** [comment]
good idea! PR welcome :)

please also check other targets/layout renderers with `IncludeMdc ` etc

**Comment by 304NotModified** [comment]
thanks! Fixed in #3050 and will be released n 4.6
